server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -795,25 +794,22 @@ private void executeFateOpsCommand(ServerContext context, FateOpsCommand fateOps
       }
     }
 
-    ReadOnlyStore<Admin> readOnlyStore = new ReadOnlyStore<>(zs);
-
     if (fateOpsCommand.print) {
       final Set<Long> sortedTxs = new TreeSet<>();
       fateOpsCommand.txList.forEach(s -> sortedTxs.add(parseTidFromUserInput(s)));
       if (!fateOpsCommand.txList.isEmpty()) {
         EnumSet<ReadOnlyTStore.TStatus> statusFilter =
             getCmdLineStatusFilters(fateOpsCommand.states);
-        admin.print(readOnlyStore, zk, zTableLocksPath, new Formatter(System.out), sortedTxs,
-            statusFilter);
+        admin.print(zs, zk, zTableLocksPath, new Formatter(System.out), sortedTxs, statusFilter);
```

Comment:
```suggestion
        admin.print((ReadOnlyTStore) zs, zk, zTableLocksPath, new Formatter(System.out), sortedTxs, statusFilter);
```
would it make sense to cast here?

Reply:
No need. The call to the method takes care of that for you.